### PR TITLE
Metric Helper Class

### PR DESCRIPTION
**Pull request by AliRKat**

### CODE REVIEW

File: Assets/Tests/PlayModeTests/HelpersTests.cs [hunk 1]
```diff
@@ -179,5 +180,60 @@ public void ConvertJsonToDictionary_withValidJson()
...
+            Assert.AreEqual(metricHelper.OSVersion, "OverNineThousand");
+            Assert.AreEqual(metricHelper.AppVersion, "5");
+
+            MetricHelper_base(metricHelper, metricValues);
+        }
+
+        [TearDown]
+        public void End()
+        {
+            Countly.Instance.ClearStorage();
```
Comment (ArtursKadikis) [inline Assets/Tests/PlayModeTests/HelpersTests.cs:235]: put this in test utility call. "TestCleanup"
call it before the test

Comment (ArtursKadikis) [reply Assets/Tests/PlayModeTests/HelpersTests.cs:235]: and after

**Review by ArtursKadikis — CHANGES_REQUESTED** [review]
Bad naming "metric helper" for metric override. 
Config should get a setter
no additions to Countly config model
no usage of "Countly.instance" within the SDK.
try adding metric helper as protected field in config, otherwise pass it to all modules

"CountlyMetricModel' should set all fields during its constructor

no comments for tests

Rework test. For now create them without initializing the SDK

**Comment by AliRKat** [comment]
[[Unity] Add metricHelper class](https://trello.com/c/cUDbRijZ/970-unity-add-metrichelper-class)

**Review by ArtursKadikis — CHANGES_REQUESTED** [review]
Eliminate metric model. Introduce custom metrics

**Pull request merged**